watcher_latency.go

```diff
@@ -14,11 +15,36 @@ func (lw *LatencyWatcher) describe(ch chan<- *prometheus.Desc) {
 }
 
 func (lw *LatencyWatcher) infoKeys() []string {
-	return nil
+	return []string{"build"}
 }
 
-func (lw *LatencyWatcher) detailKeys(rawMetrics map[string]string) []string {
```

Comment:
And This one would become "passTwoKeys". Of course this would apply to the whole interface and other watchers as well, but it's just a `Replace In All Files...` kind of affair.